HW02_A_Graph-Fails.md

```diff
@@ -0,0 +1,127 @@
+What went wrong?
+================
+Robert Gruener
+
+``` r
+knitr::opts_chunk$set(echo = TRUE, error = TRUE)
+```
+
+## HW02 Part A
+
+In this document, I will add some examples of some coding mistakes, it
+is up to you to figure out why the graphs are messing up.
+
+### First load packages
+
+It is always best to load the packages you need at the top of a script.
+It’s another common coding formatting standard (like using the
+assignment operator instead of the equals sign). In this case, it helps
+people realize what they need to install for the script and gives an
+idea of what functions will be called.
+
+It is also best coding practice to only call the packages you use, so if
+you use a package but end up tossing the code you use for it, then make
+sure to remove loading it in the first place. For example, I could use
+`library("tidyverse")` but since this script will only be using ggplot2,
+I only load ggplot2.
+
+``` r
+library("ggplot2")
+library("magrittr") #so I can do some piping
+```
+
+### Graph Fail 1
+
+What error is being thrown? How do you correct it? (hint, the error
+message tells you)
+
+``` r
+data(mpg) #this is a dataset from the ggplot2 package
+
+mpg %>% 
+   ggplot(mapping = aes(x = cty, y = hwy, color = "blue")) + 
+   geom_point() # use + rather than %>%, also cty not city
+```
+
+![](HW02_A_Graph-Fails_files/figure-gfm/unnamed-chunk-1-1.png)<!-- -->
+
+### Graph Fail 2
+
+Why aren’t the points blue? It is making me blue that the points in the
+graph aren’t blue :\`(
+
+``` r
+ggplot(data = mpg) + 
+  geom_point(mapping = aes(x = displ, y = hwy), color = "blue")
+```
+
+![](HW02_A_Graph-Fails_files/figure-gfm/unnamed-chunk-2-1.png)<!-- -->
+
+``` r
+#take out color from the aes()
+```
+
+### Graph Fail 3
+
+Two mistakes in this graph. First, I wanted to make the the points
+slightly bolder, but changing the alpha to 2 does nothing. What does
+alpha do and what does setting it to 2 do? What could be done instead if
+I want the points slightly bigger?
+
+Second, I wanted to move the legend on top of the graph since there
+aren’t any points there, putting it at approximately the point/ordered
+pair (5, 40). How do you actually do this? Also, how do you remove the
+legend title (“class”)? Finally, how would you remove the plot legend
+completely?
+
+``` r
+mpg %>% 
+ggplot() + 
+  geom_point(mapping = aes(x = displ, y = hwy, color = class), alpha = 1, size = 2) + 
+  theme(legend.direction = "horizontal") + 
+  theme(legend.position = c(.5, .90)) + #move legend to top
+  #theme(legend.position = "none") #get rid of legend completely
```

Comment:
Oh interesting, I made 4 separate graphs with the legend formatted 4 different ways, but I like that you took all his requests and put them in one graph